types/setting.go

```diff
@@ -89,6 +90,7 @@ var (
 		SettingNameStorageOverProvisioningPercentage,
 		SettingNameStorageMinimalAvailablePercentage,
 		SettingNameUpgradeChecker,
+		SettingNameCurrentLonghornVersion,
```

Comment:
NIT: perhaps we should consider having a meta/version resource, using settings for a const is sort of a hack.
But it's okay for now, since it allows us to reuse the existing implementations for settings (crd,validation,ui,etc)